docs/book/src/reference/metrics.md

```diff
@@ -73,6 +76,29 @@ for the metrics exported from the namespace where the project is running
 
 <img width="1680" alt="Screenshot 2019-10-02 at 13 07 13" src="https://user-images.githubusercontent.com/7708031/66042888-a497da80-e515-11e9-9d77-d8a9fc1159a5.png">
 
+## Default Metrics References
```

Comment:
Would it be better if we could add these default metrics in the reference section of the book (https://book.kubebuilder.io/reference/reference.html) instead of elaborating in the main page?

Reply:
Sounds good to make it appear in the reference section! Just have some updates to make an individual page `5.11.1 Metrics Reference` for the table and get it indexed in the reference. Would you like this kind of layout?